### PR TITLE
add-ShearHeatingDuretz14

### PR DESCRIPTION
**Pull request by tduretz**

### CODE REVIEW

File: SETS/ShearHeatingDuretz14.c [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      bc.value = 0.0;
+      bc.type = -1;
+    }
+  }
+  return bc;
+}
+
+SetBC SetBCT(MdoodzInstance *instance, POSITION position, double particleTemperature) {
+  SetBC     bc;
+  if (position == FREE_SURFACE) {
```
Comment (kulakovri) [inline SETS/ShearHeatingDuretz14.c]: Same value everywhere, even for internal points?
Why do we need if/else here in a first place?

File: SETS/ShearHeatingDuretz14.c [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    bc.type  = 0;
+  } else {
+    bc.value = 0.0;
+    bc.type  = 0;
+  }
+  return bc;
+}
+
+
+SetBC SetBCTNew(MdoodzInstance *instance, POSITION position, double particleTemperature) {
```
Comment (kulakovri) [inline SETS/ShearHeatingDuretz14.c:101]: Same value everywhere, even for internal points?
Why do we need if/else here in a first place?

**Review by kulakovri — CHANGES_REQUESTED** [review]

File: SETS/ShearHeatingDuretz14.c (hunk 1 above)
Comment (tduretz) [reply SETS/ShearHeatingDuretz14.c]: I've changed it. Internal points are flagged internally by the engine.

File: SETS/ShearHeatingDuretz14.c (hunk 2 above)
Comment (tduretz) [reply SETS/ShearHeatingDuretz14.c:101]: same as above

**Review by kulakovri — APPROVED** [review]

**Pull request merged**